core/storage/blog/gae_models_test.py

```diff
@@ -56,6 +56,29 @@ def setUp(self):
         self.blog_post_model.update_timestamps()
         self.blog_post_model.put()
 
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            blog_models.BlogPostModel.
+                get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.MULTIPLE_INSTANCES_PER_USER)
+
+    def test_get_export_policy(self) -> None:
+        sample_dict = {
```

Comment:
Instead of `sample_dict`, could we use a more descriptive name, for example `expected_export_policy`?

Reply:
Done!